Home Work 3

lehamagnum opened the pull request.

lehamagnum committed:
Added and configured TabBar

Added and configure clothes type scroll in CalatogVC

Added tableView, tableViewCell

lehamagnum committed:
Added and configured TabBar

lehamagnum committed:
Added and configure clothes type scroll in CalatogVC

lehamagnum committed:
Added tableView, tableViewCell

lehamagnum committed:
Merge remote-tracking branch 'origin/HomeWork3' into HomeWork3

artem7498 commented on testApp2/testApp2/ViewControllers/CatalogViewController/UIModel.swift:
```diff
@@ -0,0 +1,19 @@
+import Foundation
+import UIKit
+
+struct UIModel {
+    
+    var image: UIImage?
+    var title: String
+    var description: String
+    var price: String
+}
```
uimodel кого?

artem7498 commented on testApp2/testApp2/ViewControllers/CatalogViewController/TableViewCell.swift:
```diff
@@ -0,0 +1,147 @@
+import Foundation
+import UIKit
+
+class TableViewCell: UITableViewCell {
```
Название должно быть логичным, например, ProductCell, LegCell, CartTotalCell

artem7498 committed:
help

artem7498 committed:
help

Implements product catalog and details view

Adds a product catalog view displaying available items.
Implements navigation to a detailed product view upon user selection.
Enhances user experience by providing access to product details.

Implements product catalog and details view

Final

Final Implements product catalog and details view

artem7498 reviewed (changes requested):
Нет отступов у скролла категорий + теням отрубили что-то по бокам
По нажатию не открывается страница продукта
Ячейкам больно, если описане длинное, высота должна быть динамична




![Simulator Screenshot - iPhone 16 Pro - 2025-04-08 at 11 29 51](https://github.com/user-attachments/assets/139dee54-a84e-4135-bbfb-34c1c69e1751)
![Simulator Screenshot - iPhone 16 Pro - 2025-04-08 at 11 28 45](https://github.com/user-attachments/assets/265d8adf-bfb6-4ab7-aabc-5e9d744532c6)

artem7498 committed:
Implements product catalog and details view

Adds a product catalog view displaying available items.
Implements navigation to a detailed product view upon user selection.
Enhances user experience by providing access to product details.

lehamagnum committed:
Implements product catalog and details view

lehamagnum committed:
Final

lehamagnum committed:
Final Implements product catalog and details view

lehamagnum committed:
Merge remote-tracking branch 'origin/HomeWork3' into HomeWork3
